src/style.css

```diff
@@ -0,0 +1,34 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+h1 {
+  @apply text-6xl text-black font-extrabold;
+}
+h2 {
```

Comment:
Leave and empty line after every h1, h2... style, and do you want all your headings to be extra-bold?